packages/react-components/react-menu/src/components/Menu/Menu.test.tsx

```diff
@@ -40,7 +40,7 @@ describe('Menu', () => {
     },
   });
 
-  afterEach(() => {
+  beforeEach(() => {
```

Comment:
use `beforeEach` because a custom `afterEach` needs the user to call the `cleanup` function explicitly

Reply:
is this related to new tabster or what `cleanup` function are you reffering to ?

Reply:
`@testing-library/react` has a [cleanup](https://testing-library.com/docs/react-testing-library/api#cleanup) function which is configured by default in jest. This actually unmounts all components after the afterEach where `jest.useRealTimers` is called. The existing times that need to be cleared are still running without an explicit call of `cleanup`.

Therfore, it's just simpler to do it in `beforeEach` and let testing-libarary do its defaults

Reply:
also these timers tests should be removed since cypress now tests these interactions. I'll do that in another PR